tools/build.js

```diff
@@ -280,18 +280,20 @@ task('Build web root directory for inclusion in JAR.', [], function jarWebroot()
 
   execSync(__dirname + `/pmake.js -makers=Webroot -pom=${pom()} -builddir=${BUILD_DIR}`, {stdio: 'inherit'});
 
-  function copy(foambin) {
-    if ( fs.existsSync('./' + foambin) ) {
-      copyFile('./' + foambin, webroot + '/' + foambin);
+  if ( PACKAGE || RUN_JAR ) {
```

Comment:
Would this be easier if we just did an execSymc(`cp foam-bin-* ${webroot + '/' + foambin}`, {stdio: 'inherit'});
Maybe less portable though.
Do we clean up the old versions anywhere?

Reply:
Moved cleanup to start of genJS call. 
And suggested copy works well.